test/e2e/support/utils.js

```diff
@@ -32,31 +39,21 @@ async function gotoUrl( destUrl ) {
 	} );
 }
 
-async function login() {
-	if ( getCurrentPathName() !== '/wp-login.php' ) {
-		await gotoUrl( getUrl( 'wp-login.php' ) );
-	}
+async function goToWPPath( ...WPPaths ) {
+	await navigationTimeout( page.goto( getUrl( ...WPPaths ), { timeout: 0 } ) );
+}
 
-	await page.type( '#user_login', USERNAME );
-	await page.type( '#user_pass', PASSWORD );
+async function login() {
+	await page.type( '#user_login', WP_USERNAME );
+	await page.type( '#user_pass', WP_PASSWORD );
 	await page.click( '#wp-submit' );
-
-	await new Promise( ( resolve ) => {
-		let resolved = false;
-		const markResolvedAndResolve = () => {
-			if ( ! resolved ) {
-				resolve();
-				resolved = true;
-			}
-		};
-		setTimeout( markResolvedAndResolve, NAVIGATION_TIMEOUT );
-		page.waitForNavigation( { timeout: 0 } ).then( markResolvedAndResolve );
-	} );
+	await navigationTimeout( page.waitForNavigation( { timeout: 0 } ) );
```

Comment:
This won't work consistently in Travis. I think we should leave this as is and work on it on @aduth's separate PR see #5854

Reply:
Oh my bad, you just extracted the logic I guess :)

Reply:
Hm, yes, it's was duplicated... Not sure why there are still failing tests though.

Reply:
Hm, I don't see a difference. Those lines seem exactly the same.

Reply:
All fixed now.